src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -151,12 +162,37 @@ Test that importing different video formats into an external texture and samplin
           exp: new Uint8Array([0x00, 0xff, 0x00, 0xff]),
         }
       );
+    });
+  });
+
+g.test('importExternalTexture,destroy')
+  .desc(
+    `
+Tests that a GPUExternalTexture is destroyed by a microtask and that using it after it has been
+destroyed results in an error.
+`
+  )
+  .fn(async t => {
+    const videoUrl = getResourcePath('red-green.webmvp8.webm');
+    const video = document.createElement('video');
+    video.src = videoUrl;
+
+    let commandEncoder: GPUCommandEncoder;
+
+    await startPlayingAndWaitForVideo(video, async () => {
```

Comment:
Oops, while editing this test I realized this async function is not awaited by startPlayingAndWaitForVideo, which means the test was still not waiting for completion. (I had an error in the code that threw an exception, and it was on the console but the test was green.)